std_msgs/Int32 ID
std_msgs/Float64 x
std_msgs/Float64 y
std_msgs/Float64 vel
std_msgs/String path
std_msgs/Float64 d1
std_msgs/Float64 d2
std_msgs/Float64 origin_dist
